back/spec/services/xlsx_service_spec.rb

```diff
@@ -33,7 +33,10 @@
 
     it "contains extra columns for custom user fields" do
       custom_fields = create_list(:custom_field, 3)
-      expect(worksheet[0].cells.map(&:value)).to include *custom_fields.map(&:key)
+      custom_fields_headers = custom_fields.map do |custom_field|
+        custom_field.title_multiloc["en"]
+      end
+      expect(worksheet[0].cells.map(&:value)).to include *custom_fields_headers
```

Comment:
```suggestion
      expect(worksheet[0].cells.map(&:value)).to include(*custom_fields_headers)
```

bc Rubocop is complaining about an ambigous splat operator.